core/src/main/java/org/apache/carbondata/core/view/MVManager.java

```diff
@@ -234,7 +247,16 @@ public static String getUpdatedSegmentMap(String mergedLoadName,
   public List<MVStatusDetail> getEnabledStatusDetails() throws IOException {
     List<MVStatusDetail> statusDetails = new ArrayList<>();
     for (String database : this.getDatabases()) {
-      statusDetails.addAll(this.getEnabledStatusDetails(database));
+      try {
+        statusDetails.addAll(this.getEnabledStatusDetails(database));
```

Comment:
since already it failed to access in getScehamas, please see if this is required or not, if not please revert

Reply:
it is needed. as it try to get MV_STATUS file for all databases.